yarn-project/aztec/src/cli/cmds/start_node.ts

```diff
@@ -67,6 +67,10 @@ export const startNode = async (
     nodeConfig.disableProver = true;
   }
 
+  if (nodeConfig.disableProver && nodeConfig.disableSequencer) {
```

Comment:
Isn't this saying I can't run a node without either? Which should be fine to do?